openstudiocore/src/model/test/ElectricEquipmentITEAirCooled_GTest.cpp

```diff
@@ -264,5 +264,22 @@ TEST_F(ModelFixture, ElectricEquipmentITEAirCooled_inletportlist) {
 
 }
 
-// life cycle cost
+// test ITE approach temperature schedule
+TEST_F(ModelFixture, ElectricEquipmentITEAirCooledDefinition_approachtemperatureschedule) {
+  Model model;
+
+  ElectricEquipmentITEAirCooledDefinition definition(model);
+  ElectricEquipmentITEAirCooled electricEquipmentITEAirCooled(definition);
+  ScheduleCompact supplydeltasch(model, 5);
+  ScheduleCompact returndeltasch(model, -5);
+
+  EXPECT_EQ(7u, model.numObjects());
 
+  ASSERT_TRUE(definition.setAirFlowCalculationMethod("FlowControlWithApproachTemperatures"));
+  ASSERT_EQ("FlowControlWithApproachTemperatures", definition.airFlowCalculationMethod());
+  ASSERT_TRUE(definition.setSupplyTemperatureDifferenceSchedule(supplydeltasch));
+  ASSERT_TRUE(definition.setReturnTemperatureDifferenceSchedule(returndeltasch));
+
+  model.save(toPath("./ITE8.osm"), true);
```

Comment:
Remove this line please.

Reply:
Also, use `EXPECT_` instead of `ASSERT_` whenever you can: ASSERT should be limited to cases where it would produce a hard crash if the condition wasn't satisfied, that is before a `cast` or a `.get` on an optional.

The difference being that EXPECT will let the rest of the statements execute instead of stopping the execution altogether, so it's easier to diagnose multiple problems.

Reply:
Addressed these cosmetic changes in https://github.com/NREL/OpenStudio/pull/3781/commits/48c89a19f0146508ebe35ee1acb56da66243afa7